lib/game.rb

```diff
@@ -0,0 +1,84 @@
+class Game
+  WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+  attr_accessor :board, :player_1, :player_2
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"),board=Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def turn_count
+    self.board.cells.count{|x|x != " "}
+  end
+
+  def current_player
```

Comment:
Consider using a ternary for this method:
`@board.turn_count.even? ? @player_1 : @player_2`